ACD-416 (#6)
Actindo-AG/Documentation: RetailSuiteAccounting/Operation/11_CreateBalanceCarriedForward.md

[!!User Interface balance carried forward](../UserInterface/01_Book.md#balance-carried-forward)  
[!!User Interface settings balance carried forward](../UserInterface/02h_BalanceCarriedForward.md)  

# Create a balance carried forward

At the turn of the fiscal year, any outstanding balances must be carried forward to the next one. Closing balances will then be transferred as opening balances into the new fiscal year.

> The *Balance carried forward* menu entry of the *EDIT* context menu in the *BOOKINGS* allows to transfer outstanding account balances and open items from one fiscal year to the next one. This is possible for both personal (debtors and creditors) accounts and impersonal accounts, for example bank or tax accounts.

Reviewer: ```suggestion
By using the balance carry forward, outstanding account balances and open items can be transferred from one fiscal year to the next. This is possible for both personal accounts (debtors and creditors) and impersonal accounts (for example bank or tax accounts).
```

Special journals are preconfigured in the system to carry balances and open items forward. For example, journal 98 contains all balances carried forward for debtors open items. These settings can be completely customized. For detailed information, see [User Interface settings balance carried forward](../UserInterface/02h_BalanceCarriedForward.md).

#### Prerequisites

- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
- A new valid fiscal years is created, see [Create a fiscal year](../Integration04_ManageFiscalYear.md#create-a-fiscal-year).
- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).

> - The balance carried forward function is configured, see [User Interface settings balance carried forward](../UserInterface/02h_BalanceCarriedForward.md).

Reviewer: Ich schätze, hier brauchen wir noch nen Kapitel in Integration, wie man das konfiguriert. Im User Interface wird das nämlich nicht wirklich deutlich...

Author: Hmm, vielleicht, aber der Balance Carried Forward Konten sind eigentlich vorkonfiguriert. Man kann sie ändern/anders einstellen, aber nicht zwingend notwendig, sofern ich weiß. Ich packe es in die offenen Fragen.

#### Procedure

> *Accounting > Select fiscal year > Book > Tab BOOKINGS > Context menu EDIT > Menu entry Balance carried forward*
> 
> ![Balance carried forward](../../Assets/Screenshots/RetailSuiteAccounting/Book/BalanceCarriedForward.png "[Balance carried forward]")
> 
> 1. Click the *Balance carried forward* menu entry.  
> The *Balance carried forward: step 1* window is displayed.

Reviewer: ```suggestion
*Accounting > Select fiscal year > Book > Tab BOOKINGS*

![Balance carried forward](../../Assets/Screenshots/RetailSuiteAccounting/Book/BalanceCarriedForward.png "[Balance carried forward]")

1. Click the [EDIT] button in the header above the postings list.
A context menu is displayed.

2. Click the *Balance carried forward...* menu entry in the context menu.  
The *Balance carried forward: step 1* window is displayed.
```
Bild anpassen, Bild einfügen

  ![Balance carried forward: step 1](../../Assets/Screenshots/RetailSuiteAccounting/Book/BalanceCarriedForward01.png "[Balance carried forward: step 1]")

> 2. Select the fiscal year in the *Carry forward from FY* field.

Reviewer: ```suggestion
2. Click the *Carry forward from FY* drop-down list and select the appropriate fiscal year. All available fiscal years are displayed in the list.
```

> 3. Select the accounts to be carried forward from the *Range from - to* drop-down lists.

Reviewer: ```suggestion
3. Click the *Range from - to* drop-down lists and select the accounts to be carried forward.
```

4. Enter the booking date.

  > [Info] By default, the system displays the first day of the new fiscal year. This can be modified simply by typing in the desired booking date.

> 5. Select the *Delete old balances carried forward from selected FY* checkbox if necessary.

Reviewer: Was genau passiert dann? Wird empfohlen, das zu machen? Oder könnte das Probleme geben, weil Daten gelöscht werden?

> 

Reviewer: <img width="344" alt="Saldovortrag" src="https://user-images.githubusercontent.com/98738100/185095972-c569d96c-b381-4301-9ecd-69a7481eb235.png">
Bei mir wird noch der Punkt Kostenrechnung angezeigt. Ist das alt oder neu?

Author: Ich vermute, abhängig von deinen Einstellungen: du hast sicherlich mindestens ein Kostenstelle und einen Kostenträger erstellt?

6. Click the [CONTINUE] button.  
The *Retrieve account balances* window is displayed.

  ![Retrieve account balances](../../Assets/Screenshots/RetailSuiteAccounting/Book/RetrieveAccountBalances.png "[Retrieve account balances]")

> 7. Select the account balances to be carried forward in the *Accounts* fields.

Reviewer: ```suggestion
7. Select the account balances to be carried forward in the *Accounts* field.
```

>   > [Info] Use Ctrl and mouse click to select multiple items.

Reviewer: ```suggestion
  > [Info] To select multiple items, press and hold the **Ctrl** key while clicking with the mouse the corresponding accounts balances.
```

8. Click the [CONTINUE] button.  

> A message confirms that the balance is carried forward. The window closes automatically.

Reviewer: ```suggestion
The *Carry balances forward* window is displayed. The pop-up window *Balance carried forward completed* is displayed. The selected balances have been carried forward.

9. Click the [CLOSE] button.
The *Carry balances forward* window is closed. 
```

[!!User Interface Balance carried forward](../UserInterface/01_Book.md#balance-carried-forward)  
[!!User Interface Settings Balance carried forward](../UserInterface/02h_BalanceCarriedForward.md)  

# Create a balance carried forward

> At the turn of the fiscal year, any outstanding balances must be carried forward to the next one. Closing balances will then be transferred as opening balances into the new fiscal year.

Reviewer 2: ```suggestion
At the turn of the fiscal year, outstanding balances of the accounts of the balance sheet must be carried forward to the next one. Closing balances will then be transferred as opening balances into the new fiscal year.
```
(nur Bestandskonten werden vorgetragen, aber keine Erfolgskonten)

> By using the balance carry forward, outstanding account balances and open items can be transferred from one fiscal year to the next. This is possible for both personal accounts (debtors and creditors) and impersonal accounts (for example bank or tax accounts).

Reviewer 2: ```suggestion
By using the balance carry forward, outstanding account balances and open items are transferred from one fiscal year to the next. 
```
